src/mongo/db/exec/SConscript

```diff
@@ -148,3 +148,20 @@ env.CppUnitTest(
     ],
     NO_CRUTCH = True,
 )
+
+# env.CppUnitTest(
+#     target = "benchmark_test",
+#     source = [
+#         "benchmark_test.cpp"
+#     ],
+#     LIBDEPS = [],
+# )
+
+
+env.CppBenchmark(
+    target = "benchmark_test",
+    source = [
+        "benchmark_test.cpp"
```

Comment:
Lets move this file to src/mongo/benchmarks and rename it map_bench.cpp. You'll need to create a new Sconscript file and add it to the list of Sconscripts in src/mongo/sconscript